_includes/forms-guidance.html

```diff
@@ -17,6 +17,30 @@ <h2 id="general-accessibility-guidance-for-forms">Accessibility guidance</h2>
       ideal, from an accessibility standpoint, because of limited vision that makes it hard to scan from right to left.
     </li>
   </ul>
+  <h2 id="disabled-states-for-components-in-forms">Disabled states for components in forms</h2>
+   <ul class="usa-content-list">
+     <li><strong>Don’t use disabled states, especially for text inputs.</strong> Disabled states have low color contrast, 
+       do not offer meaningful feedback to screen reader users, and keyboard navigators can’t focus on them. 
+       These states can also be generally confusing to users. Instead, keep form elements enabled. 
+       Offer <a href="https://www.w3.org/WAI/WCAG22/Understanding/error-suggestion.html">helpful error messages</a> 
+       and inline validation to help people correct mistakes and move through the form.</li>
```

Comment:
## Shared guidance

After looking into the Form Template and Form Component page, @amyleadem found that they share content provided by this `include` file. Since these pages are identical, we only need to update in one location.

Comparing the [proposed copy](https://docs.google.com/document/d/1M1h6hLsLqhDtJb5S9NXjO4M4AKeoNjEiogtKiL59wk8/edit#heading=h.5444d1qoiuyv) (🔒) for each section, there is only one difference between the new guidance for each page. The template page guidance helper text and tooltips in addition to error messages and inline validation. 

How do we feel about adding that here knowing it'll be uniform between both template and component pages?

```suggestion
     <li><strong>Don’t use disabled states, especially for text inputs.</strong> Disabled states have low color contrast, 
       do not offer meaningful feedback to screen reader users, and keyboard navigators can’t focus on them. 
       These states can also be generally confusing to users. Instead, keep form elements enabled. 
       Offer contextual helper text, tooltips, <a href="https://www.w3.org/WAI/WCAG22/Understanding/error-suggestion.html">helpful error messages</a> 
       and inline validation to help people correct mistakes and move through the form.</li>
```

Reply:
CC: @jaclinec @annepetersen

Reply:
@mahoneycm are you saying that the copy will update as exactly the same on both the form template and form component pages (so, the copy can't be different between the two)? If so, then I guess we have to decide which one to use - the copy for the form component is much more robust which I'd advocate using, but it will make the page longer which I know is also an important thing to consider...

One good thing about using the shorter version is that it does link out to the full report which outlines when its ok to use disabled states and best practices to follow, but then we're sending users away from the page which isn't ideal. So, I guess I lean towards using the longer version. Wondering what others think.

Reply:
Hi @jaclinec , yes, the copy will update exactly on both pages. It's something we'll want to explore in future work to clean up page content. In the meantime, I'm usually a fan of keeping it simple, but the longer copy does give detailed guidance and also links out to the report if people want to go learn more.

Reply:
@jaclinec To clarify, the update to the longer version should also include the link to the [USWDS’s 2023 literature review on disabled states](https://github.com/uswds/uswds/wiki/Disabled-States-Research-Findings-2023) at the bottom of the disabled states section!